sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -1441,6 +1968,26 @@ public partial class DatasetZipDeflateCompression : Azure.Analytics.Synapse.Arti
         public DatasetZipDeflateCompression() { }
         public Azure.Analytics.Synapse.Artifacts.Models.DatasetCompressionLevel? Level { get { throw null; } set { } }
     }
+    public enum DayOfWeek
+    {
+        Sunday = 0,
+        Monday = 1,
+        Tuesday = 2,
+        Wednesday = 3,
+        Thursday = 4,
+        Friday = 5,
+        Saturday = 6,
+    }
+    public enum DaysOfWeek
+    {
+        Sunday = 0,
+        Monday = 1,
+        Tuesday = 2,
+        Wednesday = 3,
+        Thursday = 4,
+        Friday = 5,
+        Saturday = 6,
+    }
```

Comment:
Can we unify these in the swagger?  We should also look at customizing the codegen to use https://docs.microsoft.com/en-us/dotnet/api/system.dayofweek?view=netcore-3.1 if we can.

Reply:
Good catch. We will send PR and fix this soon.

Reply:
Created a PR to fix this issue: https://github.com/Azure/azure-rest-api-specs/pull/10449